zodiac-cli/src/Zodiac/Cli/TSRP/Parser.hs

```diff
@@ -38,3 +39,9 @@ requestExpiryP = option (eitherReader requestExpiryR) $
     requestExpiryR x = case parseRequestExpiry (BSC.pack x) of
       Nothing' -> Left $ "invalid request expiry: " <> x
       Just' re -> pure re
+
+lineEndingsP :: Parser LineEndings
+lineEndingsP = flag CRLF LF $
+     short 'u'
+  <> long "unix-line-endings"
+  <> help "Convert UNIX line-endings into CRLF in the provided HTTP request."
```

Comment:
I think so. The spec for the CLI was that it accepts a valid HTTP request on stdin and spits out a valid authenticated request on stdout; silently munging an invalid HTTP request (LF line endings) to make it valid without being told seems a bit broken IMO.